app.py

```diff
@@ -109,16 +107,30 @@ def serve_layout():
 )
 
 point_to_layer = assign('''function(feature, latlng, context) {
-    const {min, max, colorscale, circleOptions, colorProp} = context.props.hideout;
+    const {min, max, colorscale, circleOptions, colorProp, selectedSensor} = context.props.hideout;
     const csc = chroma.scale(colorscale).domain([min, max]);
-    circleOptions.fillColor = csc(feature.properties[colorProp]);
+
+    const extraOptions = {
```

Comment:
`extraOptions` was an attempt during debugging to eliminate "hey maybe we have a shared `circleOptions` object somehow and mutating that causes issues". That's not the case, but *not* mutating React props still seems like a Good Idea, so I left it in.